includes/routes/event/translations.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+namespace Wporg\TranslationEvents\Routes\Event;
+
+use Exception;
+use GP;
+use GP_Locales;
+use GP_Original;
+use Translation_Entry;
+use Wporg\TranslationEvents\Attendee_Repository;
+use Wporg\TranslationEvents\Event\Event_Repository_Interface;
+use Wporg\TranslationEvents\Routes\Route;
+use Wporg\TranslationEvents\Stats\Stats_Calculator;
+use Wporg\TranslationEvents\Translation_Events;
+
+/**
+ * Displays the event details page.
+ */
+class Translations_Route extends Route {
+	private $event_repository;
```

Comment:
```suggestion
	private Event_Repository_Interface $event_repository;
```

(Also requires a `use` statement)